website/docs/ckeditor-source.md

```diff
@@ -6,21 +6,30 @@ The CKEditor Source provides an abstract AtJSON source definition representing a
 
 It also provides a tool in `bin/index.ts` that can generate annotations and a concrete
 implementation of an atjson source from a given [Editor build](https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editor-Editor.html). The annotations are
-derived from the document schema in the editor. To run this tool:
+derived from the document schema in the editor. To run this tool, we recommend first copying the script bundled with this package into your local bin directory:
```

Comment:
Maybe it would be better to be more prescriptive here?

I dont think its a recommendation as much of as a hard requirement since ts-node wont run against any script in node modules

Reply:
You can, you just have to set up your ts-node config carefully.